presto-verifier/src/main/java/com/facebook/presto/verifier/PrestoVerifier.java

```diff
@@ -62,6 +62,11 @@ public int run(String[] args)
             System.setProperty("config", args[0]);
         }
 
+        if (args.length > 2) {
```

Comment:
Remove this. Logically separate changes should go in different commits, and I don't think this is how we should pass in the proxy